src/main/java/me/guichaguri/tickratechanger/TickrateContainer.java

```diff
@@ -46,6 +46,7 @@
 
     @EventHandler
     public void preInit(FMLPreInitializationEvent event) {
+		new TickrateChanger();

```

Comment:
Ok, just tried that in my dev environment and tested it in a normal MineCraft Instance, both worked pushing updates now.